billingportal/session/client.go

```diff
@@ -0,0 +1,30 @@
+// Package session provides API functions related to billing portal sessions.
+package session
+
+import (
+	"net/http"
+
+	stripe "github.com/stripe/stripe-go"
```

Comment:
Try adding `v71` to the end of this like:

``` go
    stripe "github.com/stripe/stripe-go/v71"
```

Unfortunately this is needed for all stripe-go require paths now.